contracts/priv/quartz-oem.yml

```diff
@@ -368,15 +368,15 @@ components:
           $ref: '#/components/schemas/NotificationRuleLimits'
         notificationEndpoint:
           $ref: '#/components/schemas/NotificationEndpointLimits'
-        required:
-          - orgID
-          - rate
-          - bucket
-          - task
-          - dashboard
-          - check
-          - notificationRule
-          - notificationEndpoint
```

Comment:
quartz-oem changed because OrgLimits was updated to fix a problem where required was nested under properties instead of a sibling of properties